pkg/apis/nfd/v1alpha1/nodefeaturerule/expression_test.go

```diff
@@ -0,0 +1,262 @@
+/*
+Copyright 2021 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2023 The Kubernetes Authors.
```

Reply:
Same here, no new content, just moving old one around so I'd keep the year as is.